### PR TITLE
Reduce usage of new

### PR DESCRIPTION
**Pull request by orangejenny**
@czue / @jmtroth0

### CODE REVIEW

File: corehq/apps/app_manager/static/app_manager/js/details/case_claim.js [hunk 1]
```diff
@@ -143,16 +147,11 @@ hqDefine("app_manager/js/details/case_claim", function() {
         self.blacklistedOwnerIdsExpression.subscribe(function () {
             saveButton.fire('change');
         });
-    };
 
-    var createSearchViewModel = function (searchProperties, includeClosed, defaultProperties, lang,
-        searchButtonDisplayCondition, blacklistedOwnerIdsExpression,
-        saveButton) {
-        return new searchViewModel(searchProperties, includeClosed, defaultProperties, lang,
-            searchButtonDisplayCondition, blacklistedOwnerIdsExpression, saveButton);
```
Comment (czue) [inline corehq/apps/app_manager/static/app_manager/js/details/case_claim.js]: :skull:

**Review by czue — APPROVED** [review]
@orangejenny crushing it with these changes.

**Comment by orangejenny** [comment]
ha, I hope you're enjoying reviewing all these mechanical js PRs...

**Review by jmtroth0 — APPROVED** [review]

**Comment by czue** [comment]
> ha, I hope you're enjoying reviewing all these mechanical js PRs...

the volume and cleanup along the way is impressive! :clap:

**Pull request merged**